xchange-coingi/src/main/java/org/knowm/xchange/coingi/dto/request/AuthenticatedRequestInterface.java

```diff
@@ -0,0 +1,15 @@
+package org.knowm.xchange.coingi.dto.request;
+
+public interface AuthenticatedRequestInterface {
```

Comment:
is there any point to having this interface?

Reply:
no longer